deploy/base/node/node.yaml

```diff
@@ -52,6 +55,10 @@ spec:
             - --nodeid=$(KUBE_NODE_NAME)
             - --node=true
             - --identity-provider=$(IDENTITY_PROVIDER)
+            - --metrics-endpoint=:8080
```

Comment:
The CSI driver enables the metrics collection by default, but users still need to explicitly enable the metrics on their volume spec.